packages/alfa-rules/test/sia-r66/rule.spec.tsx

```diff
@@ -262,6 +262,19 @@ test("evaluate() is inapplicable to text nodes in disabled groups", async (t) =>
   t.deepEqual(await evaluate(R66, { document }), [inapplicable(R66)]);
 });
 
+test("evaluate() is inapplicable to elements that are used in name of disabled widgets", async (t) => {
+  const document = h.document([
+    <html>
+      <fieldset name="Disabled widget name" disabled>
+        <button>Hello world</button>
+      </fieldset>
+      <div>widget name</div>
```

Comment:
```suggestion
      <fieldset aria-labelledby="label" disabled>
        <button>Hello world</button>
      </fieldset>
      <div id="label">widget name</div>
```
(but a different `<div>widget name</div>` would be applicable)